mesonbuild/interpreter.py

```diff
@@ -2211,7 +2218,16 @@ def func_subproject(self, nodes, args, kwargs):
         dirname = args[0]
         return self.do_subproject(dirname, kwargs)
 
+    def disabled_subproject(self, dirname):
+        self.subprojects[dirname] = Disabler()
+        return self.subprojects[dirname]
+
     def do_subproject(self, dirname, kwargs):
+        disabled, required, feature = extract_required_kwarg(kwargs)
+        if disabled:
+            mlog.log('\nSubproject', mlog.bold(dirname), ':', mlog.yellow('Disabled'))
```

Comment:
Better use the same message format than for other places: 'skipped: feature', mlog.bold(feature), 'disabled'

Reply:
Done.